insights/parsers/readlink_openshift_certs.py

```diff
@@ -0,0 +1,73 @@
+"""
+Get the real path of the Openshift certificate symbolic link file
```

Comment:
This doc summary line normally contains a summary of the commands/files for the parsers.  It would be better to use something like `ReadLink parsers for Openshift certificate symbolic file links` that generally describes the parsers and and the command being executed.  See other parsers in the [documentation](https://insights-core.readthedocs.io/en/latest/parsers_index.html) for examples.

Reply:
Thanks, that's a good practice！